src/bin/test.rs

```diff
@@ -98,10 +101,7 @@ pub fn execute(options: Options, config: &Config) -> CliResult<Option<()>> {
     match err {
         None => Ok(None),
         Some(err) => {
-            Err(match err.exit.as_ref().and_then(|e| e.code()) {
-                Some(i) => CliError::new("", i),
```

Comment:
I'm not really a rust veteran, but as far as I know there's no way to "downcast" a `CargoError` to a `ProcessError` .
What is the idiomatic way to do something like this?